xrtpy/response/xrt_teem.py

```diff
@@ -0,0 +1,394 @@
+        image)
+
+    data2 : array
+        XRT level1 data for second image. Shape must match data1.
+
+    binfac : integer, optional (default = 1)
+        spatial binning factor
+
+    Trange : 2 element sequence containing floats [Optional]
+        Range of log10(temperature) values to examine. Must be in order from
+        lower to higher.
+
+    Te_err_threshold : float [Optional]
+        Threshold value for the temperature error. (default = 0.5 [50%])
+        Values of temperature with error exceeding the threshold are set to 0.
+
+    photon_noise_threshold : float [Optional]
+        Threshold value for ratio of photon noise to signal. (default = 0.2
+        [20%]). If photon noise/signal exceeds this value, the corresponding
+        temperatures are set to 0.
+
+    mask : Boolean array of shape of images [Optional]
+        If provided, masks out parts of the images from the analysis. Note:
+        pixels to be masked out should be True, unmasked should be False
+
+    no_threshold : Boolean [Optional]
+        If True, no thresholds are set. (default = False)
+
+    verbose : Boolean [Optional]
+        If True, information is printed
+
+
+    Returns:
+    --------
+    Te : 2-dimensional float array
+        log10 of the derived electron temperature [K].
+    EM : 2-dimensional float array
+        log10 of the derived volume emission measure [cm^-3].
+    Terror : 2-dimensional float array
+        error of log10 temperature [K].
+    EMerror : 2-dimensional float array
+        error of log10 volume emission measure [cm^-3].
+
+    EXAMPLES:
+
+       Using this function, you can derive the coronal temperature using
+       filter ratio method.
+       >> Te, EM, Terror, EMerror = xrt_teem(hdr1, data1, hdr2, data2)
+
+       If you want to bin the image data in space to reduce photon noise, set
+       binfac to the factor by which you want to bin.  For example to bin the
+       data by a factor of 3 do:
+       >> Te, EM, Terror, EMerror = xrt_teem(hdr1, data1, hdr2, data2, binfac=3)
+       The data is binned first and then the temperature is derived. Note that
+       the image size is not reduced, but pixels within 3x3 squares are set to
+       the same value, which results from averaging over those pixels.
+
+    NOTES:
+
+       The returned values of pixels where the temperature cannot be derived
+       or the error is greater than the threshold or the photon noise is
+       greater than the threshold are set to 0. The EM for those pixels is
+       also set to 0.
+
+       The details of the coronal-temperature-diagnostic capability of
+       Hinode/XRT is described in
+       Narukage et al. 2011, Solar Phys., 269, 169.
+       http://adsabs.harvard.edu/doi/10.1007/s11207-010-9685-2
+       and
+       Narukage et al. 2013, Solar Phys.,
+       http://adsabs.harvard.edu/doi/10.1007/s11207-013-0368-7
+       These two papers are the reference papers of this program.
```

Comment:
```suggestion
    Notes
    -----
    The returned values of pixels where the temperature cannot be derived
    or the error is greater than the threshold or the photon noise is
    greater than the threshold are set to 0. The EM for those pixels is
    also set to 0.

    The details of the coronal-temperature-diagnostic capability of
    Hinode/XRT is described in
    Narukage et al. 2011, Solar Phys., 269, 169.
    http://adsabs.harvard.edu/doi/10.1007/s11207-010-9685-2
    and
    Narukage et al. 2013, Solar Phys.,
    http://adsabs.harvard.edu/doi/10.1007/s11207-013-0368-7
    These two papers are the reference papers of this program.
```
We could [include these in the bibliography](https://docs.plasmapy.org/en/latest/contributing/doc_guide.html#bibliography) too.